Zip.playground.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><Workspace version="1.0">    <FileRef location="group:Zip.playground.xcworkspace/Playground.playground/"></FileRef>    <FileRef location="group:/Users/CalQL8ed-K-OS/Workbench/open-source/Zip/./Zip.xcodeproj"></FileRef></Workspace>
```

Comment:
We can't merge this as is, since it contains absolute paths here. That won't work on other machines.

How about we just mention playground support in the README? The command you used to create it could be there as well.
Adding a playground also means that it has to be kept up to date.